File: test/data/src/py/CumulFROM.py
   
cumulTTFromZC_65 = 0
cumulTTFromZC_39 = 0
cumulTTFromZC_91 = 0     
if "Anthesis" in calendarMoments:
    if (switchMaize == 0): cumulTTFromZC_65 = cumulTT-calendarCumuls[calendarMoments.index("Anthesis")]    
if "FlagLeafLiguleJustVisible" in calendarMoments:
    if (switchMaize == 0): cumulTTFromZC_39 = cumulTT-calendarCumuls[calendarMoments.index("FlagLeafLiguleJustVisible")]  
if "EndGrainFilling"in calendarMoments:
    if (switchMaize == 0): cumulTTFromZC_91 = cumulTT-calendarCumuls[calendarMoments.index("FlagLeafLiguleJustVisible")]

File: test/data/usecase/src/cs/CumulFROM.cs

double cumulTTFromZC_65 = 0;
double cumulTTFromZC_39 = 0;
double cumulTTFromZC_91 = 0;
        
if calendarMoments.Contains("Anthesis"){
    if (SwitchMaize == 0)
        cumulTTFromZC_65 = cumulTT-calendarCumuls[calendarMoments.IndexOf("Anthesis")];
}    
if calendarMoments.Contains("FlagLeafLiguleJustVisible"){
    if (SwitchMaize == 0)
        cumulTTFromZC_39 = cumulTT-calendarCumuls[calendarMoments.IndexOf("FlagLeafLiguleJustVisible")];
}   
if calendarMoments.Contains("EndGrainFilling"){
    if (SwitchMaize == 0)
        cumulTTFromZC_91 = cumulTT-calendarCumuls[calendarMoments.IndexOf("FlagLeafLiguleJustVisible")];
}